flake.nix

```diff
@@ -0,0 +1,137 @@
+{
+  inputs = {
+    nixpkgs_.url = "github:deemp/flakes?dir=source-flake/nixpkgs";
+    nixpkgs.follows = "nixpkgs_/nixpkgs";
+    my-codium.url = "github:deemp/flakes?dir=codium";
+    drv-tools.url = "github:deemp/flakes?dir=drv-tools";
+    flake-utils_.url = "github:deemp/flakes?dir=source-flake/flake-utils";
+    flake-utils.follows = "flake-utils_/flake-utils";
+    haskell-tools.url = "github:deemp/flakes?dir=language-tools/haskell";
+    devshell.url = "github:deemp/flakes?dir=devshell";
+    my-lima.url = "github:deemp/flakes?dir=lima";
```

Comment:
As mentioned above, I think you can drop this dependency, among others, if you either use .lhs for the docs or .md + markdown-unlit.